tests/rust_usage_test/tests/integration_test.rs

```diff
@@ -1879,7 +1893,10 @@ mod follow_impls {
         ];
         let tab = <flatbuffers::ForwardsUOffset<flatbuffers::Table>>::follow(&buf[..], 0);
         assert_eq!(tab.get::<flatbuffers::ForwardsUOffset<&str>>(fi2fo(0), Some("abc")), Some("abc"));
-        assert_eq!(tab.get::<flatbuffers::ForwardsUOffset<&[u8]>>(fi2fo(0), Some(&vec![70, 71, 72][..])), Some(&vec![70, 71, 72][..]));
+        #[cfg(target_endian = "little")]
```

Comment:
@tymcauley Just to be safe, want to add an assert here for using a Vector to access the u8 data? For both endiannesses.

Reply:
I was thinking about that, but don't the assertions on the next few lines accomplish the same thing?

Reply:
Indeed they do!